src/lib.rs

```diff
@@ -140,38 +157,56 @@ This is a mean initialization method based on the [kmeans++](https://en.wikipedi
 initialization algorithm.
 */
 #[cfg(not(feature = "parallel"))]
-fn initialize_plusplus<V: Value>(data: &ArrayView2<V>, k: usize, seed: Option<u128>) -> Array2<V> {
+fn initialize_plusplus<V: Value>(
+    data: &ArrayView2<V>,
+    k: usize,
+    n_trials: Option<usize>,
+    seed: Option<u128>,
+) -> Array2<V> {
     assert!(k > 1);
     assert!(data.dim().0 > 0);
+    let n_trials = n_trials.unwrap_or(2 + (k as f64).ln().floor() as usize);
```

Comment:
I have issues with the default value as it doesn't preserve the original behavior of this algorithm.